homeassistant/components/sensor/__init__.py

```diff
@@ -302,7 +306,7 @@ def _default_to_device_class_name(self) -> bool:
         """
         return self.device_class not in (None, SensorDeviceClass.ENUM)
 
-    @property
```

Comment:
I didn't do `last_reset` yet ...  That probably shouldn't go in the description though?